index.html

```diff
@@ -6,15 +6,15 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
- [ ] Kindly add the in your Pull Request :point_down:  just copy and past it there

![pr](https://user-images.githubusercontent.com/57562869/150215907-66492daf-5f4a-4abf-bbb0-1934360434f4.png)